CHANGES.rst

```diff
@@ -1,6 +1,335 @@
 Traits CHANGELOG
 ================
 
+Release 6.0.0rc0
+----------------
+
+XXX reST markup for mentioned objects, to give links into docs?
+XXX Double check that all PRs are mentioned
+
+Released: XXXX-XX-XX
```

Comment:
To be edited in later, once we have a better idea of actual release date (likely tomorrow or Thursday).